### ISSUE TITLE
[findrepe][0.1.0] Exception with an option but no directories

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jareens/softenido#2]
```
findrepe 0.1.0

What steps will reproduce the problem?

run this

java -jar FindRepe.jar -n

you get this exception

Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: 0
    at com.softenido.findrepe.FindRepe.<init>(FindRepe.java:86)
    at com.softenido.findrepe.Main.main(Main.java:168)

it should give an advice about no directories specified

```

Original issue reported on code.google.com by `flik...@gmail.com` on 4 Apr 2009 at 8:44

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.1.1
```

Original comment by `flik...@gmail.com` on 4 Apr 2009 at 9:22
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
verified
```

Original comment by `flik...@gmail.com` on 4 Apr 2009 at 9:23
- Changed state: **Verified**
